.github/workflows/webui-periodic.yml

```diff
@@ -1,4 +1,4 @@
-# ======================================
+    # ======================================
```

Comment:
This looks like an accidental change?

Reply:
Yes, something did not work right :-/ That also fails the test.